Sources/FoundationEssentials/Calendar/Calendar_Gregorian.swift

```diff
@@ -38,7 +38,7 @@ extension Date {
         timeIntervalSinceReferenceDate / 86400 + Self.julianDayAtDateReference
     }
 
-    func julianDay() throws (GregorianCalendarError) -> Int {
+    package func julianDay() throws (GregorianCalendarError) -> Int {
```

Comment:
I don't see uses of this, the initializer below, or the error type from our tests, was this an accidental addition?

Reply:
It’s used here: https://github.com/swiftlang/swift-foundation/blob/3179457ef5a0c1f6a07ad042c192012d35a0b81f/Tests/FoundationEssentialsTests/GregorianCalendarTests.swift#L69

Reply:
Ah yeah that's probably why I missed it in my search since it didn't actually call the function - ok thanks for the explanation